src/dune_util/log.ml

```diff
@@ -40,16 +40,26 @@ let init ?(file = File.Default) () =
 let init_disabled () = Fdecl.set t None
 let t () = Fdecl.get t
 
+let log_oc oc msg =
```

Comment:
If you switch the order of these args you won't have to write a lambda later twice.